api/core/v1alpha2/virtual_machine.go

```diff
@@ -159,58 +233,79 @@ const (
 )
 
 type VirtualMachineStatus struct {
-	Phase                   MachinePhase                             `json:"phase"`
-	Node                    string                                   `json:"nodeName"`
-	VirtualMachineIPAddress string                                   `json:"virtualMachineIPAddressName"`
-	IPAddress               string                                   `json:"ipAddress"`
-	BlockDeviceRefs         []BlockDeviceStatusRef                   `json:"blockDeviceRefs"`
-	GuestOSInfo             virtv1.VirtualMachineInstanceGuestOSInfo `json:"guestOSInfo"`
-	Conditions              []metav1.Condition                       `json:"conditions,omitempty"`
-	Stats                   *VirtualMachineStats                     `json:"stats,omitempty"`
-	MigrationState          *VirtualMachineMigrationState            `json:"migrationState,omitempty"`
-	ObservedGeneration      int64                                    `json:"observedGeneration,omitempty"`
-	// RestartAwaitingChanges holds operations to be manually approved
-	// before applying to the virtual machine spec.
-	//
+	Phase MachinePhase `json:"phase"`
+	// The name of the node on which the VM is currently running.
+	Node string `json:"nodeName"`
+	// Name of `virtualMachineIPAddressName` holding the ip address of the VirtualMachine.
+	VirtualMachineIPAddress string `json:"virtualMachineIPAddressName"`
+	// IP address of VM.
+	IPAddress string `json:"ipAddress"`
+	// The list of attached block device attachments.
+	BlockDeviceRefs []BlockDeviceStatusRef                   `json:"blockDeviceRefs,omitempty"`
+	GuestOSInfo     virtv1.VirtualMachineInstanceGuestOSInfo `json:"guestOSInfo,omitempty"`
+	// Specific points in VM's runtime.
+	Conditions []metav1.Condition `json:"conditions,omitempty"`
+	// VirtualMachine statistics.
+	Stats *VirtualMachineStats `json:"stats,omitempty"`
+	// Migration info.
+	MigrationState *VirtualMachineMigrationState `json:"migrationState,omitempty"`
+	// Generating a resource that was last processed by the controller.
+	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
+	// RestartAwaitingChanges holds operations to be manually approved before applying to the virtual machine spec.
 	// Change operation has these fields:
-	//
-	//	operation enum(add|remove|replace)
-	//	path string
-	//	currentValue any (bool|int|string|struct|array of structs)
-	//	desiredValue any (bool|int|string|struct|array of structs)
-	//
+	// * operation enum(add|remove|replace)
+	// * path string
+	// * currentValue any (bool|int|string|struct|array of structs)
+	// * desiredValue any (bool|int|string|struct|array of structs)
 	// Such 'any' type can't be described using the OpenAPI v3 schema.
 	// The workaround is to declare a whole change operation structure
 	// using 'type: object' and 'x-kubernetes-preserve-fields: true'.
```

Comment:
This comment should not be a part of CRD. It is internal information for developers, not for users.